.rubocop.yml

```diff
@@ -681,3 +685,149 @@ Style/IfUnlessModifierOfIfUnless:
 Style/ZeroLengthPredicate:
   Enabled: false
 
+# DISABLED - not useful
+Bundler/OrderedGems:
+  Enabled: false
+
+# do we require the encoding magic comments any more
+# DISABLED - TODO (2 entries)
+Layout/EmptyLineAfterMagicComment:
+  Enabled: false
+
+# extra new lines
+# DISABLED - TODO (2 entries)
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
+# extra new lines
+# DISABLED - TODO (23 entries)
+Layout/EmptyLinesAroundExceptionHandlingKeywords:
+  Enabled: false
+
+# DISABLED - not useful
```

Comment:
All the `DISABLED - not useful` comments seem redundant. I'm a little leary of putting the number of entries in their as well (I assume that number will change shortly after this is merged and then be wrong). The `TODO PUP-8003` notes seem valuable - the ticket might also be a good spot to list the number of entries at this given date/revision for evaluating how much work each ticket is.

Reply:
I wanted to call out the ones that I had reviewed and decided where not useful to consider adding back versus the large number of them that are complete unknowns as to why they are disabled.

Reply:
> versus the large number of them that are complete unknowns as to why they are disabled

I think that "these are disabled because they aren't useful to us/applicable to our style guide" is a valid default assumption for all the disabled checks. Which is why I consider that comment redundant. I would only put in comments when that isn't the case.